xla/service/gpu/fusions/fusions.cc

```diff
@@ -142,15 +142,17 @@ absl::StatusOr<std::unique_ptr<FusionInterface>> GetFusionEmitter(
                          ->config()
                          .debug_options();
   auto check_mlir_emitters = [&](bool check = true) {
+    if (!check) return false;
+
     if (!opts.xla_gpu_enable_mlir_emitters()) {
       return false;
     }
-    CHECK(!check ||
-          mlir_converter::IsHloConversionSupported(
-              analysis.fusion(),
-              fusion_info.analysis().device_info().gpu_compute_capability()))
-        << "Unsupported fusion: "
-        << analysis.fusion_root(0).instruction().parent()->ToString();
+    if (!mlir_converter::IsHloConversionSupported(analysis.fusion(),
```

Comment:
Let's add a separate flag like `xla_gpu_enable_mlir_emitters`, but to allow the fallback to legacy emitters. `xla_gpu_enable_mlir_emitter_fallback`?